script/build-fuzzers

```diff
@@ -6,30 +6,30 @@ if [[ "$(uname -s)" != Linux ]]; then
   exit 1
 fi
 
-if [[ -z "$LIB_FUZZER_PATH" ]]; then
-  echo "LIB_FUZZER_PATH not set"
-  exit 1
-fi
-
-CC=${CC:-clang}
-CXX=${CXX:-clang++}
-LINK=${LINK:-clang++}
+CC=clang
+CXX=clang++
```

Comment:
Don't remove possibility to override from env

Reply:
right, was debugging why clang wasnt selected by the makefile